### ISSUE TITLE
projectos - vazio

### ISSUE THREAD

**Issue by dncpax** [ulmi/ulmi.github.io#9]
devemos ter responsáveis por cada secção que necessita de manutenção

**Comment by ulmi** [comment]
Sim...eu já tenho 2 para colocar, mas resolvi por assim...falta é texto a explicar como se pode adicionar os projectos
